recipe/build.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -ex
+
+# necessary for cross-compilation to point to the right env
+export PKG_CONFIG_PATH=$PREFIX/lib/pkgconfig
+
+# HACK: extend $CONDA_PREFIX/meson_cross_file that's created in
+# https://github.com/conda-forge/ctng-compiler-activation-feedstock/blob/main/recipe/activate-gcc.sh
+# https://github.com/conda-forge/clang-compiler-activation-feedstock/blob/main/recipe/activate-clang.sh
+# to use host python; requires that [binaries] section is last in meson_cross_file
+echo "python = '${PREFIX}/bin/python'" >> ${CONDA_PREFIX}/meson_cross_file.txt
+
+# meson-python already sets up a -Dbuildtype=release argument to meson, so
+# we need to strip --buildtype out of MESON_ARGS or fail due to redundancy
+MESON_ARGS_REDUCED="$(echo $MESON_ARGS | sed 's/--buildtype release //g')"
+
+mkdir builddir
+# -wnx flags mean: --wheel --no-isolation --skip-dependency-check
+$PYTHON -m build -w -n -x \
+        -Cbuilddir=builddir \
+        -Csetup-args=${MESON_ARGS_REDUCED// / -Csetup-args=} \
```

Comment:
I don't understand this line. I am not familiar with the forward slash syntax in that context. What do the do?

Reply:
This add -Csetup-args= in front of every arguments in `MESON_ARGS_REDUCED`. This must be an advanced bash variable replacement

```
❯ echo $MESON_ARGS_REDUCED
--prefix=/home/conda/feedstock_root/build_artifacts/scikit-learn_1709907407136/prefix -Dlibdir=lib

```
❯ echo ${MESON_ARGS_REDUCED// / -Csetup-args=}
--prefix=/home/conda/feedstock_root/build_artifacts/scikit-learn_1709907407136/prefix -Csetup-args=-Dlibdir=lib
````